baselines/Google Chat Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -300,3 +300,72 @@ Agencies SHOULD configure DLP rules to block or warn on sharing files with sensi
 9.  Select the appropriate action to warn or block sharing, based upon the agency's individual requirements.
 10. In the **Alerting** section, choose a severity level, and optionally, check **Send to alert center to trigger notifications**.
 11. Review the rule details, mark the rule as **Active**, and click **Create.**
+
+## 7. Content Reporting
+
+This section covers the content reporting functionality.
+
+### Policies
+
+#### GWS.CHAT.7.1v0.1
+Chat Content Reporting SHALL be Enabled.
+
+- Rationale
+  - This ensures that users are able to report content for different categories such as sensitive or confidential information. This provides additional monitoring capabilities.
+- Last Modified: February 13, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - None
+
+#### GWS.CHAT.7.2v0.1
+All Conversation Types SHALL be Selected.
+
+- Rationale
+  - This ensures that users are able to report content for different categories such as sensitive or confidential information. This provides additional monitoring capabilities.
+- Last Modified: February 13, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - None
+
+  #### GWS.CHAT.7.3v0.1
+Spam, Confidential Information, and Sensitive Information SHALL be Selected.
+
+- Rationale
+  - This ensures that users are able to report content for different categories such as sensitive or confidential information. This provides additional monitoring capabilities.
+- Last Modified: February 13, 2024
+- Notes
+  - The other reporting categories can be implemented depending on agency needs.
+
+- MITRE ATT&CK TTP Mapping
+  - None
+
+### Resources
+
+- [Set-up content moderation for Chat](https://apps.google.com/supportwidget/articlehome?hl=en&article_url=https%3A%2F%2Fsupport.google.com%2Fa%2Fanswer%2F13471510%3Fhl%3Den&assistant_id=generic-unu&product_context=13471510&product_name=UnuFlow&trigger_context=a)
+
+### Prerequisites
+
+-   None
```

Comment:
Chat history must be on for users to report messages, recommend noting that here. https://apps.google.com/supportwidget/articlehome?article_url=https%3A%2F%2Fsupport.google.com%2Fa%2Fanswer%2F13471510%3Fvisit_id%3D638439737035840473-729100435&assistant_id=generic-unu&product_context=13471510&product_name=UnuFlow&trigger_context=a

Reply:
Added this.